WORKFLOW.md

```diff
@@ -0,0 +1,170 @@
+# Everything below can run unmodified.
+BRANCH_NAME="hotfix/$NEW_VERSION"
+git checkout production
+git pull origin
+git checkout -b "$BRANCH_NAME"
+```
+
+After creating the hotfix branch, fix that bug, create a changelog fragment and
+commit the changes in the hotfix branch!
+
+Next, proceed to the [Merging release branches](#merging-release-branches) section.
+
+## Merging release branches
+
+**NOTE**:
+The steps in this document must be performed in a release or hotfix branch.
+See the
+[Preparing a feature release](#preparing-a-feature-release)
+or
+[Preparing a hotfix release](#preparing-a-hotfix-release)
+section for steps to create a release or hotfix branch.
+
+After creating a release or hotfix branch, you must follow these steps to merge
+the branch to `production` and `main`:
+
+1. On the branch that is to be released, prepare the code and documentation for
+   release.
+   1. Bump the version.
+        - If the release is a backwards compatible change use 
+          ``poetry version patch``
+        - If the release is non-backwards compatible, use 
+          ``poetry version minor``
+   2. Bump copyright years as appropriate.
+   3. Collect changelog fragments as appropriate.
+   4. Run unit/integration/CI/doc tests as appropriate.
+   5. Commit all changes to git.
+
+2. Push the branch to GitHub.
+
+3. Create a new pull request to merge to `production`.
+   1. Select `production` as the "base" merge branch.
+   2. Select the release or hotfix branch as the "compare" merge branch.
+   3. Wait for CI test results (and approvals, when possible).
+      1. It is the release engineer's discretion to ask for and require PR
+      approvals. A release branch will usually contain code that has already 
+      been reviewed, unless it is a hotfix. If the release is a hotfix, it is
+      recommended to get approvals.
+
+      > WARNING: **Merge conflicts**
+      >
+      > Merge conflicts halt the release process when merging to `production`
+      > unless it is a trivial conflict (like the "version" in `pyproject.toml`).
+   4. Merge the branch to `production`. Do not delete the branch!
+
+4. Create a new tag and a new release.
+   1. Click on the "Releases" section. Then click "Draft a new release".
+   2. Click the "Choose a tag" dropdown, type the new version, and press Enter.
+   2. Select `production` as the target branch.
+   3. Type the new version as the release title.
+   4. Paste the changelog as the release description.
+   5. Click "Publish release" to publish the new tag and release on GitHub.
+
+5. Publish the release
+   1. Documentation updates will be pulled direction from the `production`
+      branch. If the release is purely documentation, there are no further steps
+      to publishing.
+   2. Code updates need to be published to PyPI. To do so, the following steps
+      will need to be performed on a Terminal window:
+      1. git checkout production
+      2. git pull --tags
+      3. poetry publish --build
```

Comment:
I think GH Markdown supports a code block even within this context:

## Code

```markdown
   2. Code updates need to be published to PyPI. To do so, the following steps
      will need to be performed on a Terminal window:

      ```shell
      git checkout production
      git pull --tags
      poetry publish --build
      ```
```

## Result

2. Code updates need to be published to PyPI. To do so, the following steps
   will need to be performed on a Terminal window:

   ```shell
   git checkout production
   git pull --tags
   poetry publish --build
   ```

Reply:
Let me try that out and see how it gets rendered ....

Reply:
You are 100% right about that! I feel like I've been learning a lot of markdown in reviews lately so thank you for that!